File: Selenium/Activities/Activity3.py
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium import webdriver
from selenium.webdriver.common.by import By


services=FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=services) as driver:
    driver.get("https://training-support.net")
    print("Page title :- " ,driver.title)
    driver.find_element(By.XPATH,"//a[@id='about-link']").click()
    print("Page title :- " ,driver.title)

    driver.get("https://www.training-support.net/selenium/login-form")
    print("Page Title:- " + driver.title)

    driver.find_element(By.XPATH,"//input[@id='username']").send_keys("admin")
    driver.find_element(By.XPATH,"//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH,"//button[text()='Log in']").click
    driver.close()

File: Selenium/Activities/Activity3.java
package example;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity3 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/login-form/");
		System.out.println("Title of the page is: " + driver.getTitle());
		driver.findElement(By.xpath("//input[@name='username']")).sendKeys("admin");
		driver.findElement(By.xpath("//input[@name='password']")).sendKeys("password");
		driver.findElement(By.xpath("//button[text()='Submit']")).click();
		driver.quit();

	}

}
